### DISCUSSION TITLE
Trying to add http headers | X-Frame-Options | X-XSS-Protection | Upgrade-Insecure-Requests

### DISCUSSION THREAD

**Discussion by daniel8800** [drakkan/sftpgo#1258] [Q&A]
Hello, sorry to bother but I'm trying to add those http headers (in httpd) but I don't know if I can do that or how can I do it.
The headers I'm trying to add are the following:

X-XSS-Protection: 1; mode=block
X-Frame-Options: deny
Upgrade-Insecure-Requests: 1

Hope you can help me with this question.

Regards.

**Comment by ajaykumar-gbi** [comment]
+1

**Reply by drakkan** [reply]
Check the docs about the security headers you can set
